src/Octoshift/Services/GithubApi.cs

```diff
@@ -1184,29 +1184,31 @@ private static GithubSecretScanningAlert BuildSecretScanningAlert(JToken secretA
             Secret = (string)secretAlert["secret"],
         };
 
-    private static GithubSecretScanningAlertLocation BuildSecretScanningAlertLocation(JToken alertLocation) =>
-        new()
+    private static GithubSecretScanningAlertLocation BuildSecretScanningAlertLocation(JToken alertLocation)
+    {
+        var details = alertLocation["details"];
+        return new GithubSecretScanningAlertLocation
         {
             LocationType = (string)alertLocation["type"],
-            Path = (string)alertLocation["details"]["path"],
-            StartLine = (int)alertLocation["details"]["start_line"],
-            EndLine = (int)alertLocation["details"]["end_line"],
-            StartColumn = (int)alertLocation["details"]["start_column"],
-            EndColumn = (int)alertLocation["details"]["end_column"],
-            BlobSha = (string)alertLocation["details"]["blob_sha"],
-            IssueTitleUrl = (string)alertLocation["details"]["issue_title_url"],
-            IssueBodyUrl = (string)alertLocation["details"]["issue_body_url"],
-            IssueCommentUrl = (string)alertLocation["details"]["issue_comment_url"],
-            DiscussionTitleUrl = (string)alertLocation["details"]["discussion_title_url"],
-            DiscussionBodyUrl = (string)alertLocation["details"]["discussion_body_url"],
-            DiscussionCommentUrl = (string)alertLocation["details"]["discussion_comment_url"],
-            PullRequestTitleUrl = (string)alertLocation["details"]["pull_request_title_url"],
-            PullRequestBodyUrl = (string)alertLocation["details"]["pull_request_body_url"],
-            PullRequestCommentUrl = (string)alertLocation["details"]["pull_request_comment_url"],
-            PullRequestReviewUrl = (string)alertLocation["details"]["pull_request_review_url"],
-            PullRequestReviewCommentUrl = (string)alertLocation["details"]["pull_request_review_comment_url"],
+            Path = (string)details?["path"],
```

Comment:
I'm not sure if we want to use a null-conditional here. To me it's an exceptional case and probably should not happen (we don't expect it to happen). Is there a scenario that the `details` would be `null` ?